### ISSUE TITLE
Shift+Space key to maximize docked editor window like native unity windows

### ISSUE THREAD

**Issue by JustusPan** [UnitySuperEditor/SuperEditor#27]
Could you please consider to add this shortcut to the code editor？
My Unity version is 2020.3.24LTS, and this shortcut could not work properly.

**Comment by UnitySuperEditor** [comment]
Yes, there will be a shortcut panel in the next version, where you can customize various shortcuts
